samples/drivers/espi/prj_npcx7m6fb_evb.conf

```diff
@@ -0,0 +1,15 @@
+# eSPI + npcx7m6fb_evb
+CONFIG_LOG=y
+CONFIG_LOG_BUFFER_SIZE=4096
+CONFIG_LOG_PROCESS_THREAD_SLEEP_MS=100
+<<<<<<< Updated upstream
```

Comment:
Unmerged problem

Reply:
Oops. I fixed it in the other CL. Will change the order of this.